solidity/contracts/token/HypERC20Collateral.sol

```diff
@@ -63,6 +65,20 @@ contract HypERC20Collateral is FungibleTokenRouter, MovableCollateralRouter {
         return wrappedToken.balanceOf(_account);
     }
 
+    function quoteTransferRemote(
+        uint32 _destinationDomain,
+        bytes32 _recipient,
+        uint256 _amount
+    ) external view virtual override returns (Quote[] memory quotes) {
+        quotes = new Quote[](2);
+        quotes[0] = _quoteTransferRemote(
```

Comment:
What if a child class returns multiple quotes? IMO this cannot make assumptions here that it is only one quote

Reply:
I also think this is all the more reason to document this better. I would even go a step further and say that we should name this internal function differently. It would be too easy for consumers to believe that they have to be quoting for the wrappedTokenAmount in here.

I would maybe even question whether we should have an internal function that is being overridden at all. Wdyt about simply having a default implementaiton here and in HypNative that uses an internal function to quote from the postDispatch hook, but if a child overrides quoting, they have to explicitly override the external function, and not the internal function (which imo makes it too hard to reason about how that interplays with subclasses)